test/rewards/AutoCompoundingRewards.ts

```diff
@@ -11,7 +11,12 @@ import Contracts, {
     TestPoolCollection,
     TestRewardsMath
 } from '../../components/Contracts';
-import { EXP2_INPUT_TOO_HIGH, RewardsDistributionType, ZERO_ADDRESS } from '../../utils/Constants';
+import {
```

Comment:
I noticed now that this wasn't added to the `Profile.ts`. We should have a few test cases there as well

Reply:
done